content/03.prior-knowledge.md

```diff
@@ -1,3 +1,41 @@
 ### Techniques that build on prior knowledge and indirectly related data are necessary for many rare disease applications
+
 >This section will highlight promising approaches for analyzing rare disease data to extract biological insights. 
 >We will discuss techniques like transfer learning, representation learning, cascade learning, integrative analysis, and knowledge-graph creation and use that leverage other knowledge and data sources to construct testable hypotheses from rare diseases datasets with limited sample sizes 1–8.
+
+#### Representation learning
+
+Representation learning, also called feature learning, is the process of learning features from raw data, where a feature is an individual variable or property.
+An algorithm or approach will construct features as part of training and, in the case of supervised feature learning, use those features to predict labels on input data. 
+Using an example from transcriptomics, an unsupervised method such as matrix factorization can be used to extract a low-dimensional representation of the gene-level data, learning features that are a combination of input genes' expression levels [@doi:10.1093/bioinformatics/btq503; @doi:10.1186/s13059-020-02021-3].
+Low-dimensional representations trained on compendia of transcriptomic data can then be used as input to supervised machine learning methods [@doi:10.1186/s12859-020-3427-8]. 
+Supervised neural networks used in medical imaging studies [@doi:10.1016/j.procs.2016.07.014] (reviewed in [@doi:10.1016/j.zemedi.2018.11.002] and [@doi:10.1098/rsif.2017.0387]), which are trained to predict labels or classes, are also an example of representation learning.
+Learned features in the medical imaging domain may be a series of edges that constitute a blood vessel. 
```

Comment:
Line 13 and 14 seem to be a bit abrupt after line 12. It may be helpful to add some context to smoothen the flow.